<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversor</class>
 <widget class="QMainWindow" name="Conversor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>502</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1273</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conversor de Números</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/icons/favicon.ico</normaloff>:/icon/icons/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="baseLayout">
     <item>
      <layout class="QVBoxLayout" name="inputLayout">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Entrada:</string>
         </property>
         <property name="buddy">
          <cstring>inputBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="inputBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="analizeBtn">
       <property name="text">
        <string>Analizar y Traducir</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="btn_lex">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lexLayout">
       <item>
        <widget class="QLabel" name="lexLabel">
         <property name="text">
          <string>Analizador Léxico</string>
         </property>
         <property name="buddy">
          <cstring>lexTable</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="lexTable">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>555</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="lex_sin">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sinLayout">
       <item>
        <widget class="QLabel" name="sinLabel">
         <property name="text">
          <string>Analizador Sintáctico</string>
         </property>
         <property name="buddy">
          <cstring>sinTable</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="sinTable">
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="treeBtn">
         <property name="text">
          <string>Ver Árbol</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="separator"/>
    <addaction name="actionImportar"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionGramatica"/>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuMen"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionImportar">
   <property name="text">
    <string>Importar archivo</string>
   </property>
   <property name="toolTip">
    <string>Importar archivo a la entrada</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGramatica">
   <property name="text">
    <string>Gramática</string>
   </property>
   <property name="iconText">
    <string>Gramática</string>
   </property>
   <property name="toolTip">
    <string>Gramática utilizada</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="toolTip">
    <string>Acerca del grupo</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
